src/modules/postScore.js

```diff
@@ -0,0 +1,25 @@
+const postScore = async (playerScore) => {
+  try {
+    const response = await fetch(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Vd1Kn67VwbjHPDwYt8cS/scores/',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/Json',
+          charset: 'utf-8',
+        },
+        body: JSON.stringify(playerScore),
+      },
+    );
+
+    if (!response.ok) {
+      return false;
+    }
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    return false;
+  }
+};
+
+export default postScore;
```

Comment:
As per the project requirements, after posting a score, the leader board should not refresh automatically. Instead, we need to hit the refresh button to re-fetch the scores from the API and then based on that data, update our UI. However, in your case, the UI refreshes when we submit the score. Kindly fix this issue.